experimental/examples/attributable-map-demo/package.json

```diff
@@ -0,0 +1,84 @@
+{
+	"name": "@fluid-example/attributable-map-demo",
```

Comment:
Same here, no need for the word "demo" in the package name, it's already implied by the scope.